src/data/showcases.js

```diff
@@ -927,10 +927,10 @@ export const Showcases = [
     title: "Liqwid",
     description:
       "Liqwid is a non-custodial pooled lending protocol with liquid staking built on Cardano.",
-    preview: require("./showcase/liqwid.png"),
+    preview: require("./showcase/liqwid.webp"),
     website: "https://liqwid.finance/",
     source: null,
-    tags: ["lending", "token", "marketplace", "nftsupport"],
+    tags: ["lending", "token", "marketplace", "nftsupport", "token"],
```

Comment:
```suggestion
    tags: ["lending", "token", "marketplace", "nftsupport"],
```

Reply:
@rcheiko you will have to commit this before we go any further (please do this from the GitHub UI so we can track these edits).